Tests/aweXpect.Reflection.Tests/ThatType.Has.AttributeTests.cs

```diff
@@ -147,7 +147,69 @@ private class FooChildClass2 : FooClass2
 			{
 			}
 		}
-		
+
```

Comment:
The negated tests for the following files are still missing:
- `ThatType.IsNotARecordStruct`
- `ThatType.IsNotAStruct`
- `ThatType.IsNotAbstract`
- `ThatType.IsNotAnEnum`
- `ThatType.IsNotAnInterface`
- `ThatType.IsNotGeneric`
- `ThatType.IsNotInternal`
- `ThatType.IsNotNested`
- `ThatType.IsNotPrivateProtected`
- `ThatType.IsNotPrivate`
- `ThatType.IsNotProtectedInternal`
- `ThatType.IsNotProtected`
- `ThatType.IsNotPublic`
- `ThatType.IsNotSealed`
- `ThatType.IsPrivateProtected`
- `ThatType.IsPrivate`
- `ThatType.IsProtectedInternal`
- `ThatType.IsProtected`